lib/index.js

```diff
@@ -3,7 +3,7 @@ import fs from 'node:fs'
 import cli_progress from 'cli-progress'
 import fetch from 'node-fetch'
 import fastq from 'fastq'
-import cheerio from 'cheerio'
```

Comment:
Can you remove the unnecessary `;` at the end?

Reply:
Done